components/ads130e08/.eil.yml

```diff
@@ -0,0 +1,17 @@
+---
+components:
+  - name: ads130e08
+    description: Driver for ADS130E08 ADC
+    group: adc-dac
+    groups: []
+    code_owners: weslleymfd
+    depends:
+    thread_safe: yes
+    targets:
+      - name: esp32
```

Comment:
do you have ESP32C3 or other chips to test? with changes to the pin assignment, it should work. if you don't, something like "should work on other ESP32" in the doc would be nice.

Reply:
I don't have other ESP32 chips to test. I'll change the doc and say it may be supported for other chips.